use-timescale/backup-restore.md

```diff
@@ -0,0 +1,113 @@
+products: [cloud]
+keywords: [backups, restore]
+tags: [recovery, failures]
+---
+
+# Back up and recover your $SERVICE_SHORTs
+
+$CLOUD_LONG automatically handles backup for your $SERVICE_LONGs using the `pgBackRest` tool. You don't need to perform backups manually. What's more, with [cross-region backup][cross-region], you are protected even if an entire AWS region goes down.
+
+$CLOUD_LONG automatically creates one full backup every week, and
+incremental backups every day in the same region as your $SERVICE_SHORT. Additionally, all WAL ([Write-Ahead Log][wal])
+files are retained back to the oldest full backup. This means that you always
+have a full backup available for the current and previous week: 
+
+![Backup in Tiger Cloud](https://assets.timescale.com/docs/images/database-backup-recovery.png)
+
+In the event of a storage failure, a $SERVICE_SHORT automatically recovers from backup
+to the point of failure. If the whole availability zone goes down, your $SERVICE_LONGs are recovered in a different zone. In the event of a user error, where a point-in-time recovery needs to be done, you can [create a point-in-time recovery fork][create-fork].
+
+## Enable cross-region backup 
+
+<Availability products={['cloud']} price_plans={['enterprise']} />
+
+For added reliability, you can enable cross-region backup. In this case, you have two identical backups of your $SERVICE_SHORT at any time, but one of them is in a different AWS region.
+
+You enable cross-region backup when creating a $SERVICE_SHORT, or configure it for an existing $SERVICE_SHORT in $CONSOLE_LONG:
+
+<Procedure>
+
+1. In [$CONSOLE_SHORT][console], select your $SERVICE_SHORT and click `Operations` > `Backup & restore`. 
+
+1. In `Cross-region backup`, select the region in the dropdown and click `Enable backup`. 
+
+   ![Create cross-region backup](https://assets.timescale.com/docs/images/tiger-cloud-console/create-cross-region-backup-tiger-cloud.png)
+
+   You can now see the backup, its region, and creation date in a list. 
+
+</Procedure>
+
+You can have one cross-region backup per $SERVICE_SHORT. To change the region of your backup:
+
+<Procedure>
+
+1. In [$CONSOLE_SHORT][console], select your $SERVICE_SHORT and click `Operations` > `Backup & restore`.
+
+1. Click the trash icon next to the existing backup to disable it. 
+
+   ![Disable cross-region backup](https://assets.timescale.com/docs/images/tiger-cloud-console/cross-region-backup-list-tiger-cloud.png)
+
+1. Create a new backup in a different region. 
+
+</Procedure>
+
+
+## Create a point-in-time recovery fork
+
+You recover your $SERVICE_SHORT from a destructive or unwanted action by creating a point-in-time recovery fork. You can recover a $SERVICE_SHORT to any point within the period [defined by your pricing plan][pricing-and-account-management]. The original $SERVICE_SHORT stays untouched to avoid losing data created since the time of recovery.
```

Comment:
```suggestion
To recover your $SERVICE_SHORT from a destructive or unwanted action, create a point-in-time recovery fork. You can recover a $SERVICE_SHORT to any point within the period [defined by your pricing plan][pricing-and-account-management]. The original $SERVICE_SHORT stays untouched to avoid losing data created since the time of recovery.
```